packages/extensions/openapi-to-typespec/src/emiters/emit-main.ts

```diff
@@ -37,18 +38,19 @@ function getHeaders() {
 }
 
 function getServiceInformation(program: TypespecProgram) {
+  const { modules, namespaces: namespacesSet } = getMainImports(program);
   const imports = [
-    `import "@typespec/rest";`,
-    `import "@typespec/http";`,
-    `import "./routes.tsp";`,
-    ``,
-    `using TypeSpec.Rest;`,
-    `using TypeSpec.Http;`,
-    `using TypeSpec.Versioning;`,
-  ];
-  const content = generateServiceInformation(program);
-
-  return [...imports, content].join("\n");
+    ...new Set<string>([`import "@typespec/rest";`, `import "@typespec/http";`, ...modules, `import "./routes.tsp"`]),
```

Comment:
why do we need to wrap a set of string within the array?
And since we have `join("\n")` in L53, why do we need to have them here?

Reply:
When get modules from getMainImports, the implementation already uses `Set<string>` to deal with the duplication removal.

Reply:
Changed the inner `Set` to `Array`.